dev/codesniffer/ruleset.xml

```diff
@@ -3,11 +3,16 @@
 <ruleset name="Dolibarr">
 	<description>Dolibarr coding standard.</description>
 
-	<exclude-pattern>*/conf.php</exclude-pattern>
-	<exclude-pattern>*/includes/*</exclude-pattern>
-	<exclude-pattern>*/documents/*</exclude-pattern>
-	<exclude-pattern>*/dev/vagrant/*</exclude-pattern>
-	<exclude-pattern>*/custom/*</exclude-pattern>
+	<exclude-pattern>build/html</exclude-pattern>
+	<!--
```

Comment:
As explained, any option containing the string "documents" is buggy. I don't know why…
There's no php or js file to check there anyway so it should not be a major issue.